tests/Makefile

```diff
@@ -94,6 +94,8 @@ git:
 
 define GIT_INIT
   git init && echo '*.sh text eol=lf'> .gitattributes && $(if $1,touch $1 && )\
+	git add -A && git config --local user.name 'OPAM test environment' &&\
+	git config --local user.email noreply@ocaml.org && git commit -m "Initial commit"
```

Comment:
Thanks for the fix!
Just a thing, `git add` & `git commit` are duplicated, shouldn't be
```bash
git config --local user.name 'OPAM test environment' &&\
git config --local user.email noreply@ocaml.org &&\
git add -A && git commit -m "Initial commit"
```

Reply:
Yes indeed, and it was the cause for the build failure (I failed to copy-paste code from my Debian patch to the upstream Opam repo...)